dask_cuda/disk_io.py

```diff
@@ -164,11 +164,16 @@ def disk_write(path: str, frames: Iterable, shared_filesystem: bool, gds=False)
     cuda_frames = tuple(hasattr(f, "__cuda_array_interface__") for f in frames)
     frame_lengths = tuple(map(nbytes, frames))
     if gds and any(cuda_frames):
-        import cucim.clara.filesystem as cucim_fs
+        import kvikio
 
-        with cucim_fs.open(path, "w") as f:
+        futs = []
+        with kvikio.CuFile(path, "w") as f:
             for frame, length in zip(frames, frame_lengths):
-                f.pwrite(buf=frame, count=length, file_offset=0, buf_offset=0)
+                futs.append(
+                    f.pwrite(buf=frame, count=length, file_offset=0, buf_offset=0)
+                )
+            for each_fut in futs:
+                futs.get()
```

Comment:
Currently we need to call `get` on each of these. Maybe this could be handled by a `wait` function that does that for us. Raised issue ( https://github.com/rapidsai/kvikio/issues/79 )